.pre-commit-config.yaml

```diff
@@ -1,28 +1,30 @@
 ---
 minimum_pre_commit_version: 2.9.2
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: trailing-whitespace
+        exclude: ^vendor/|^tests/.*/fixtures/.*|^tests/integration/test_data/.*
```

Comment:
Note for follow-up PR: `^tests/.*/fixtures/.*` and `^vendor/` doesn't seem to exist, probably something that's been removed.

I think it'd make sense to put `^tests/integration/test_data/.*` in a global exclude pattern, as we probably don't want any linting at all to happen there.